modules/TerraformController.py

```diff
@@ -196,11 +197,11 @@ def dump_attack_data(self, dump_name):
         folder = "attack_data/" + dump_name
         os.mkdir(folder)
 
-        servers = []
-        if self.config['windows_domain_controller'] == '1':
-            servers.append('windows_domain_controller')
-        if self.config['windows_server'] == '1':
-            servers.append('windows_server')
+        servers = ['splunk_server']
+        # if self.config['windows_domain_controller'] == '1':
+        #     servers.append('windows_domain_controller')
+        # if self.config['windows_server'] == '1':
+        #     servers.append('windows_server')
```

Comment:
This will go uncommented. Just have it here because I was testing.

- [ ] Remove comments